builtin/providers/aws/resource_aws_autoscaling_group.go

```diff
@@ -412,6 +426,11 @@ func resourceAwsAutoscalingGroupRead(d *schema.ResourceData, meta interface{}) e
 	d.Set("health_check_type", g.HealthCheckType)
 	d.Set("launch_configuration", g.LaunchConfigurationName)
 	d.Set("load_balancers", flattenStringList(g.LoadBalancerNames))
+	if g.SuspendedProcesses != nil {
```

Comment:
If we have suspended processes, and they are removed (assuming they can be removed) externally, this guard will prevent us from detecting the drift, correct? If so, then we should remove this guard and just make sure `flattenAsgSuspendedProcesses` will return an empty list when `g.SuspendedProcesses` is nil